### DISCUSSION TITLE
Interested in Calyx Research

### DISCUSSION THREAD

**Discussion by anthonyabeo** [cucapra/futil#1836] [Introductions]
Hello Everyone,

My name is Anthony and I am interested in getting involved with Calyx research.

I completed an undergraduate Computer Science program a few years ago and have recently developed an interest in compilers. Before that, I spent time learning all I could about computer architecture so a project like Calyx will be right up my alley.

I have read the Calyx paper and understand at a high level what the project aims to do, but I could use some recommendations and guidance on where to start contributing.

Thank you.

**Comment by rachitnigam** [comment]
hey @anthonyabeo! Thanks for your interest: I'd recommend looking at the issues [marked as "good first issues"](https://github.com/calyxir/calyx/labels/good%20first%20issue) and seeing what interests you. You can comment on one of the issues and we can see if it would be a good fit to work on!

**Reply by sampsyo** [reply]
Indeed, welcome, @anthonyabeo! If you're interested, one of our plans for 2024 is also to open up our weekly technical meetings so anyone from the community to attend. Once the logistics for that are in place, it would be great if you'd like to listen in on those!

**Reply by anthonyabeo** [reply]
Thanks @rachitnigam. I'll make sure to do that.

**Reply by anthonyabeo** [reply]
Sure, @sampsyo that would be nice. Looking forward to those.
